glean-core/pings.yaml

```diff
@@ -17,6 +17,7 @@ baseline:
     The `baseline` ping is automatically sent when the application is moved to
     the background.
   include_client_id: true
+  send_if_empty: true
```

Comment:
Would we need data review for this addition?

Reply:
That's a good question for @chutten

Reply:
To my understanding the original data reviews cover sending the "baseline" ping whether it is empty or not, at rates up to the stated frequency. No data review needed in my mind.